src/redux/error/error.reducer.js

```diff
@@ -0,0 +1,18 @@
+import { SET_ERROR } from './error.types';
+
+const initialState = {
+  error: null
+};
+
+const errorReducer = (action, state = initialState) => {
```

Comment:
it seems that there must be another order(state,=>action)

Reply:
Thanks for your attention, that's true but then we got this code smell.
![image](https://user-images.githubusercontent.com/31392756/100146117-b1a22680-2e99-11eb-89a5-200133f9eb18.png)

Reply:
you can assign empty object to avoid this warning

Reply:
thank you, didn't know about that

Reply:
@VitalikVoicix @nadiyafomenko  fixed it, thank you